api/serializers.py

```diff
@@ -20,10 +20,26 @@ class PatientSerializer(serializers.ModelSerializer):
     class Meta:
         model = Patient
         fields = '__all__'
-    
+
     def create(self, validated_data):
         user_data = validated_data.pop('user')
         user = User.objects.create_user(**user_data)
         patient = Patient.objects.create(**validated_data, user=user)
         patient.save()
         return patient
+
+
+class HealthOfficerSerializer(serializers.ModelSerializer):
+    user = UserSerializer()
+    url = serializers.URLField(source='get_absolute_url', read_only=True)
+
+    class Meta:
+        model = HealthOfficer
+        fields = '__all__'
+
+    def create(self, validated_data):
+        user_data = validated_data.pop('user')
+        user = User.objects.create_user(**user_data)
+        officer = HealthOfficer.objects.create(**validated_data, user=user)
+        patient.save()
```

Comment:
Why call **patient.save()** here? think it should be **officer.save()**.

Reply:
can you see that i have made changes @austinoski

Reply:
I'll check it now.